src/Modal/Modal.tsx

```diff
@@ -199,18 +203,11 @@ export const Modal: React.FC<Props> = ({
                 velocity: 8,
               },
             }}
-            onClick={
-              // Ignore the type of `event` because the React.MouseEvent
-              // generic will be a union of 170 different DOM elements;
-              // TypeScript can't handle that.
-              //
-              // @ts-ignore
```

Comment:
This works fine now?

Reply:
Yes. We're widening `MotionComponent`'s type to be `React.ComponentType<MotionProps>` instead of the union of all motion components so that we can get the typing we need without overloading TypeScript with a union of 170 DOM elements.